wdl/AoU_Mitochondria.wdl

```diff
@@ -0,0 +1,126 @@
+version 1.0
+
+
+##########################################################################################
```

Comment:
I guess this is carried over from the other workflows, but I find these not good practice anymore given that we have `meta` section for workflows and tasks.

Reply:
fixed